python/nx-cugraph/README.md

```diff
@@ -89,48 +89,106 @@ interface to its CUDA-based graph analytics library) and
+| complete_graph                       | 23.12                    |
+| complete_multipartite_graph          | 23.12                    |
+| connected_components                 | 23.12                    |
+| core_number                          | 24.02                    |
+| cubical_graph                        | 23.12                    |
+| cycle_graph                          | 23.12                    |
+| davis_southern_women_graph           | 23.12                    |
+| degree_centrality                    | 23.12                    |
+| desargues_graph                      | 23.12                    |
+| descendants                          | 24.02                    |
+| descendants_at_distance              | 24.02                    |
+| diamond_graph                        | 23.12                    |
+| dodecahedral_graph                   | 23.12                    |
+| edge_betweenness_centrality          | 23.10                    |
+| eigenvector_centrality               | 23.12                    |
+| empty_graph                          | 23.12                    |
+| florentine_families_graph            | 23.12                    |
+| from_pandas_edgelist                 | 23.12                    |
+| from_scipy_sparse_array              | 23.12                    |
+| frucht_graph                         | 23.12                    |
+| generic_bfs_edges                    | 24.02                    |
+| heawood_graph                        | 23.12                    |
+| hits                                 | 23.12                    |
+| house_graph                          | 23.12                    |
+| house_x_graph                        | 23.12                    |
+| icosahedral_graph                    | 23.12                    |
+| in_degree_centrality                 | 23.12                    |
+| is_arborescence                      | 24.02                    |
+| is_bipartite                         | 24.02                    |
+| is_branching                         | 24.02                    |
+| is_connected                         | 23.12                    |
+| is_forest                            | 24.02                    |
+| is_isolate                           | 23.10                    |
+| is_strongly_connected                | 24.02                    |
+| is_tree                              | 24.02                    |
+| is_weakly_connected                  | 24.02                    |
+| isolates                             | 23.10                    |
+| k_truss                              | 23.12                    |
+| karate_club_graph                    | 23.12                    |
+| katz_centrality                      | 23.12                    |
+| krackhardt_kite_graph                | 23.12                    |
+| ladder_graph                         | 23.12                    |
+| leiden                               | ?                        |
```

Comment:
Is it worth listing functions we don't yet implement? I would rather we only include ones we currently have.

Reply:
There was feedback that having the future functions included was good for generating interest, but maybe it's better to do that when we have a known future version they'll be a part of, or have them in a separate table.  I removed them in the latest commit.